manifests/mysqld_exporter.pp

```diff
@@ -104,6 +94,14 @@
   String $package_name,
   String $user,
   String $version,
+
+  Stdlib::Absolutepath              $cnf_config_path = '/etc/.my.cnf',
+  Stdlib::Host                      $cnf_host        = localhost,
+  Stdlib::Port                      $cnf_port        = 3306,
+  String[1]                         $cnf_user        = login,
+  Variant[Sensitive[String],String] $cnf_password    = 'password',
```

Comment:
can you update this?
```suggestion
  Variant[Sensitive[String[1]],String[1]] $cnf_password    = 'password',
```

Reply:
An empty password is probably valid though?